integration-tests/src/test/java/org/apache/fineract/integrationtests/ExternalBusinessEventTest.java

```diff
@@ -265,55 +265,48 @@ private void logBusinessEvents(List<ExternalEventDTO> allExternalEvents) {
         });
     }
 
-    public void verifyBusinessEvents(BusinessEvent... businessEvents) {
-        List<ExternalEventDTO> allExternalEvents = ExternalEventHelper.getAllExternalEvents(requestSpec, responseSpec);
+    public void verifyBusinessEvents(final BusinessEvent... businessEvents) {
+        final var allExternalEvents = ExternalEventHelper.getAllExternalEvents(requestSpec, responseSpec);
```

Comment:
Not a big fan of var, what's the reason of changing these?

Reply:
Using var has become a habit for me from working on another long-term project. I see only one drawback: the code review process in GitHub can get a bit complicated. I plan to follow the existing coding style of this project. So should I stop using var altogether? Are there specific cases where it’s allowed or prohibited?

Reply:
@oleksii-novikov-onix I wouldn't say it's prohibited, it's just not appearing right now in the codebase and I'd keep it out just for consistency.